packages/engine/src/avatar/AnimationManager.ts

```diff
@@ -10,39 +10,13 @@ import { findRootBone, processRootAnimation } from './animation/Util'
 import avatarBoneMatching, { findSkinnedMeshes, makeBindPose } from './AvatarBoneMatching'
 import { makeDefaultSkinnedMesh } from './functions/avatarFunctions'
 
-export interface ikTargets {
-  rightHandTarget: Object3D
-  leftHandTarget: Object3D
-  rightFootTarget: Object3D
-  leftFootTarget: Object3D
-}
-
 //Create all IK targets as object 3ds, stored in
 //a named struct and in an object 3d hierarchy
 //the former allows easy accessability while the
 //latter allows for threejs keyframe animation
 export const animationManager = defineState({
```

Comment:
Please capitalize

Reply:
Got it, I still get occasionally tripped up in figuring when to use camelcase or pascal